[DEVX-161] Improve finance api docs (#118)
iZettle/api-documentation: finance-api/user-guides/fetch-payout-info.md

<!-- Is there any limit for how oldest transactions can be fetched? -->  

* [Prerequisites](#prerequisites)
* [Fetch payout information](#fetch-payout-information)
* [Related task](#related-task)
* [Related API reference](#related-api-reference)

## Prerequisites
* Make sure that authorization is set up with the required OAuth scope using [Authorization OAuth2 API](../../authorization.adoc). 
<!-- to be continued if any -->

## Fetch payout information
You can fetch payout information from a merchant's Zettle account for any given day.

Fetch payout information on a specific day. If you don't specify `at`, you will get the latest payout information by default.
     
   ```
   GET /organizations/self/payout-info?at={when_to_fetch_payout}
   ```

   Example:
   
   The following example fetches the payout information for 7 June, 2021. The next payout amount will be 6605.89 kr.
   
   Request   
   ```
   GET /organizations/self/payout-info?at=2021-06-07
   ```
   Response   
   ```json
      {
           "data": {
               "totalBalance": 660589,
               "currencyId": "SEK",
               "nextPayoutAmount": 660589,
               "discountRemaining": 0,

>                "periodicity": "DAILY"

Reviewer: There seems to different currencies used in different example.  `GBP` was used for this example `/organizations/self/accounts/preliminary/balance`

I think it's better to stick to one currency for all the examples and maybe best to use `GBP` since it's widely known internationally when compared to SEK.

Reviewer: <img width="1042" alt="Screenshot 2021-07-07 at 12 27 45" src="https://user-images.githubusercontent.com/18021829/124744180-c7db8f00-df1e-11eb-8591-ba8631fc7eeb.png">

Here also SEK being used.

Fetch payout information
===

> Using the Finance API, you can fetch payout information from a merchant's Zettle account. By default, it's fetched from the merchant's Zettle liquid account. The payout information includes the account balance, currency, payout, periodicity, and so on. 

Reviewer 2: > By default, it's fetched from the merchant's Zettle liquid account.

We don't do payouts from preliminary account. We don't need this sentence.

Reviewer 2: Better to say "payout amount" when listing all the attributes.
 Is "so on" - `discountRemaining`?  If yes why not just put it there?

<!-- Is there any limit for how oldest transactions can be fetched? -->  

* [Prerequisites](#prerequisites)
* [Fetch payout information](#fetch-payout-information)
* [Related task](#related-task)
* [Related API reference](#related-api-reference)

## Prerequisites
* Make sure that authorization is set up with the required OAuth scope using [Authorization OAuth2 API](../../authorization.adoc). 
<!-- to be continued if any -->

## Fetch payout information
You can fetch payout information from a merchant's Zettle account for any given day.

> 1. Fetch payout information on a specific day. If you don't specify `at`, you will get the latest payout information by default.

Reviewer 2: Do we have a step 2 somewhere? If not `1.` is not necessary here.

     
   ```

>    GET /organizations/self/payout-info?at={time}

Reviewer 2: On this row we are interpollating {time} and in the example below only date specified.
This makes it unclear at what time payout-info returned is specified.

   ```

   Example:
   
   The following example fetches the payout information for 7 June, 2021.
   
   Request   
   ```
   GET /organizations/self/payout-info?at=2021-06-07
   ```
       
   Response

    ```json
    {
        "data": {
            "totalBalance": 9022,
            "currencyId": "SEK",

>             "nextPayoutAmount": 0,

Reviewer 2: This is a weird example (probably with minimum account value set by the merchant).
Total balance is `90.22 SEK` but `nextPayoutAmount=0`

Fetch payout information
===
Use the Finance API to fetch payout information from a merchant's account. The payout information includes the account balance, currency, payout, periodicity, and remaining discounts. 

A payout is to deposit the account balance to the merchant's bank account or their PayPal Wallet for PayPal users at the [scheduled time](https://www.zettle.com/help/articles/1084784-deposits). The scheduled time varies depending on the country.

If the merchant's configuration has a minimum account balance, then the payout will deposit the account balance minus the minimum account balance.

* [Prerequisites](#prerequisites)

> * [Fetch payout information](#fetch-payout-information)

Reviewer 3: ```suggestion
* [Fetch payout information](#fetch-payout-information-1)
```

* [Prerequisites](#prerequisites)
* [Step 1: Fetch payout information](#step-1-fetch-payout-information)
* [Related task](#related-task)
* [Related API reference](#related-api-reference)

## Prerequisites
* Make sure that authorization is set up using [Authorization OAuth2 API](../../authorization.adoc). 
<!-- to be continued if any -->

## Step 1: Fetch payout information
You can fetch payout information from a merchant's Zettle account for any given day.

1. Optional: Fetch your organisation UUID. 
   > **Tip:** You don't need to fetch the organisation UUID, as you can specify the organisation UUID as `self` in requests that require it.

    ```
    GET users/me
    ```
   Example:
       
   The following example response returns organisation UUID `ab305d54-75b4-431b-adb2-eb6b9e546013`.

    ```
    {
        "uuid": "de305d54-75b4-431b-adb2-eb6b9e546014",
        "organizationUuid": "ab305d54-75b4-431b-adb2-eb6b9e546013"
    }
    ```

2. Fetch payout information on a specific day. If you don't specify `at`, you will get the latest payout information by default.
      > **Tip:** You specify the organisation UUID as `self`.
     
   ```

>    GET /organizations/{organizationUuid}/accounts/liquid/payout-info?{at}

Reviewer 2: Where does this come from? There is no such endpoint in finance service

    ```
    GET users/me
    ```
   Example:
       
   The following example response returns organisation UUID `ab305d54-75b4-431b-adb2-eb6b9e546013`.

    ```
    {
        "uuid": "de305d54-75b4-431b-adb2-eb6b9e546014",
        "organizationUuid": "ab305d54-75b4-431b-adb2-eb6b9e546013"
    }
    ```

2. Fetch payout information on a specific day. If you don't specify `at`, you will get the latest payout information by default.
      > **Tip:** You specify the organisation UUID as `self`.
     
   ```
   GET /organizations/{organizationUuid}/accounts/liquid/payout-info?{at}
   ```

   Example:
   
   The following example request fetches the payout information for 7 June, 2021.
   
   ```
   GET /organizations/self/payout-info?at=2021-06-07
   ```
       
   The following example response returns the payout information for 7 June, 2021.

    ```json
    {
        "data": {
            "totalBalance": 9022,
            "currencyId": "SEK",
            "nextPayoutAmount": 0,
            "discountRemaining": 10000,
            "periodicity": "WEEKLY"
        }
    }
    ```

>    > **Tip**: The periodicity can be changed to `daily` or `monthly` in the merchant's Zettle Back Office > **Account & settings** > **Scheduled deposits**.                                                                                   

Reviewer 3: hmmm from an **integrator** perspective I am not sure if makes sense we mention that the periodicity can be changed (because the merchant is the owner of this config). Maybe rephrase this or remove it?

```suggestion
   > **Tip**: The periodicity can be `daily`, `weekly` or `monthly` according to the merchant's configuration at Zettle Back Office > **Account & settings** > **Scheduled deposits**.                                                                                   
```
